sf-stream/src/runtime.rs

```diff
@@ -109,80 +93,83 @@ impl SfStreamer {
 
         Self {
             context,
-            current_version: version,
             resolver,
-            block_height: block_info.block_height,
-            current_epoch: epoch,
+            current_block_height: block_event.height(),
+            current_epoch: block_event.epoch(),
             mp_sender: mp_client_sender,
         }
     }
 
     pub async fn start(&mut self) {
+        println!("DMLOG INIT aptos v1");
         loop {
-            self.batch_convert(100).await;
+            self.convert_next_block().await;
         }
     }
 
-    pub async fn batch_convert(&mut self, batch_size: u16) -> Vec<extractor::Transaction> {
-        let mut result: Vec<extractor::Transaction> = vec![];
-        match &self.context.db.get_first_txn_version() {
-            Ok(version_result) => match version_result {
-                Some(oldest_version) => {
-                    if oldest_version > &self.current_version {
-                        panic!(
-                            "[sf-stream] oldest txn version is {} but requested version is {}",
-                            oldest_version, &self.current_version
-                        );
-                    }
-                }
-                None => {
-                    return vec![];
-                }
-            },
+    pub async fn convert_next_block(&mut self) -> Vec<TransactionPB> {
```

Comment:
This is going to do a Lot more non-grouped reads than the previous implementation

Reply:
What do you mean by this? We are still batch querying?